commit.go

```diff
@@ -124,15 +127,16 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
 
 	img, err := is.Transport.GetStoreImage(b.store, dest)
 	if err == nil {
+		imgID = img.ID
```

Comment:
@mheon, made a comment in the prior rev just before you posted this rev.  I'd pref a check to see if the error is unknownimage and then error out if not and I did in my example.  Your call.

Reply:
@TomSweeneyRedHat I think that's a good option, let me make the change and repush